design2.css

```diff
@@ -0,0 +1,136 @@
+body {
+	background-color: #008800;
+	font-family: Verdana, Arial, Helvetica, sans-serif;
+	text-align: center;
+}
+* {
+	margin: 0px;
+	padding: 0px;
+}
+html {height:100%; margin-bottom:1px;} /* Forces vertical scroll bar to be visible */
+#wrapper {
+	width: 900px;
+	margin-right: auto;
+	margin-left: auto;
+	background-image: url(images/Column1.jpg);
+	background-repeat: repeat-y;
+	text-align: left;
+	background-color: #FFFFFF;
+}
+#mainphoto {
+	background-image: url(images/Main_photo1.jpg);

```

Comment:
A clearer name for `images/Main_photo1.jpg` would probably help with readability of the code. Perhaps assign the string to a more descriptive variable name and pass that as the url argument function instead